src/main/scala/com/ambiata/poacher/hdfs/Hdfs.scala

```diff
@@ -170,6 +170,9 @@ object Hdfs {
     })
   } yield a
 
+  def readBytes(p: Path, chunksize: Int = Streams.DefaultChunkSize): Hdfs[Array[Byte]] =
```

Comment:
Unused default arg. Reinforces why we shouldn't do this :)  Given the changes coming down (i.e. https://github.com/ambiata/poacher/compare/topic/path#diff-b04ff8df1fc8bb69319615cccb969153R120) I would just remove the argument and accept a fixed chunk size for now (this default arg is being removed from mundane on same branch).